### PR TITLE
Db add keyword

### PR DESCRIPTION
**Pull request by iandalrymple**
1) Added the title to the URL table
2) Added the keyword table 
3) Updated stored procedures to accomodate 1 and 2 (lot of work)
4) Updated the EER Visio file to show the current schema and SP's
5) Updated the dbApi file in lib folder to accomodate the changed stored procedures 
6) Updated all test stored procedures on database 
7) Included a back up of DB in SQL folder just in case

### CODE REVIEW

**Comment by nigelrobbins3** [comment]
It looks like all the tests were pulled out. Was that just because they were not working with the new structure? No pressure to add them back, testing isn't in our requirements.

**Comment by iandalrymple** [comment]
Wait what do you mean - I am confused and worried I messed something up.
please elaborate

File: SQL/TestingScratchpad.sql [hunk 1]
```diff
@@ -1,35 +1,20 @@
...
-SELECT @resLocal; SET @resLocal = -99;
-
-CALL wcSp_InsertIntoTempEdges("http://15", "http://16", 11, @resLocal);
-SELECT @resLocal; SET @resLocal = -99;
-
-CALL wcSp_InsertNewTree("http://newCookie", "RogerPedacker", 11, "BFS", 22, @resLocal);
-SELECT @resLocal; SET @resLocal = -99;
-
-DELETE FROM wc_TempEdges;
-SET @url1 = CONCAT('http://', CONCAT(CAST(unix_timestamp() AS char), '1'));
```
Comment (nigelrobbins3) [inline SQL/TestingScratchpad.sql:27]: I mean that all these tests were removed. I was just wondering why, but it looks like you added a bunch of javascript tests instead.

**Comment by iandalrymple** [comment]
Oh ok cool - yeah that was just my scratchpad for messing around and
debugging. I create TEST stored procedures for every use case stored
procedure and then those are consumed by the dbTestApi.

Ian

**Pull request merged**